avalon/tools/projectmanager/widget.py

```diff
@@ -180,6 +185,16 @@ def __init__(self, silo=None, parent=None):
     def set_silo(self, silo, refresh=True):
         """Set the root path to the ItemType root."""
         self._silo = silo
+        try:
+            self.silo_asset = io.find_one(
+                {'$and': [
```

Comment:
This doesn't need to be an "$and" query, right? Couldn't this be just one regular query?
```python
io.find_one({
    'type': 'asset',
     'name': silo,
     'silo': None
})
```

Reply:
Hmm, could you set up a reproducable example? It should work.

Also, the `try... except`  shouldn't be needed since `find_one()` will return None if there's no match. Unless you're expecting to pass specific values to the function that will break the query - but I wouldn't know why that would happen?

Reply:
You're absolutely right... just realized what was the reason why it didn't work correctly when I tried it. That was also the reason why I put that part into `try...except.` Now, I see! Thanks